features/step_definitions/cluster_configuration/cluster_config.rb

```diff
@@ -33,3 +33,41 @@
   end
 end
 
+# helper method to enabled qe-app-registry
+Given /^I create #{QUOTED} catalogsource for my#{OPT_QUOTED} cluster$/ do |catalog_name, ocp_version|
+  ensure_admin_tagged
+  ensure_destructive_tagged
+
+  step %Q/I store master major version in the :master_version clipboard/ unless ocp_version
+  ocp_version ||= cb.master_version
+  project("openshift-marketplace")
+  iib='quay.io/openshift-qe-optional-operators/ocp4-index:latest'
+  case ocp_version
+  when '4.5stage'
+    iib='registry-proxy.engineering.redhat.com/rh-osbs/iib-pub-pending:v4.5'
+  when '4.5', '4.5qe'
+    iib = 'quay.io/openshift-qe-optional-operators/qe45-index:latest'
+  when  '4.6stage'
+    iib = 'registry-proxy.engineering.redhat.com/rh-osbs/iib-pub-pending:v4.6'
+  when '4.7stage'
+    iib='registry-proxy.engineering.redhat.com/rh-osbs/iib-pub-pending:v4.7'
+  when '4.8', '4.7', '4.7qe', '4.6', '4.6qe'
+    iib='quay.io/openshift-qe-optional-operators/ocp4-index:latest'
+  else
+    iib='quay.io/openshift-qe-optional-operators/ocp4-index:latest'
+  end
```

Comment:
/cc @anpingli  for review.

Reply:
@liangxia this is based on @anpingli's shell script https://gitlab.cee.redhat.com/aosqe/aosqe-tools/-/blob/master/app_registry_tools/create_index_catalogsource.sh

Reply:
How about use two options   index_version and test_type.
  
  if test_type=stage and index_version > v4.4
      iib=registry-proxy.engineering.redhat.com/rh-osbs/iib-pub-pending:v${index_version}'
  else 
     if index_version in (v4.1, v4.2,v4.3,v4.4,v4.5)
        iib=quay.io/openshift-qe-optional-operators/qe45-index:latest
     else
        iib=quay.io/openshift-qe-optional-operators/ocp4-index:latest
  fi

Reply:
@anpingli PR updated, PTAL again.  thx